src/aliasthis.d

```diff
@@ -24,17 +24,17 @@ import ddmd.root.outbuffer;
 import ddmd.tokens;
 import ddmd.visitor;
 
-/**************************************************************/
+/***********************************************************
+ * alias ident this;
+ */
 extern (C++) final class AliasThis : Dsymbol
 {
 public:
-    // alias Identifier this;
```

Comment:
Why remove this comment? I like to have comments like these that relate source syntax with the corresponding data structure.